test/integration/serverTest.js

```diff
@@ -1,29 +1,58 @@
 const chai = require('chai')
 const chaiHttp = require('chai-http')
 const sinonChai = require('sinon-chai')
+const chaiDateTime = require('chai-datetime')
 
 const expect = chai.expect
 const { after, afterEach, before, beforeEach, describe, it } = require('mocha')
 const sinon = require('sinon')
 const { helpers } = require('brave-alert-lib')
-const imports = require('../../index.js')
+const { sleep } = require('brave-alert-lib/lib/helpers')
+const imports = require('../../index')
 
 const db = imports.db
 const redis = imports.redis
 const server = imports.server
-const XETHRU_STATE = require('../../SessionStateXethruEnum.js')
-
-const MOV_THRESHOLD = 17
+const braveAlerter = imports.braveAlerter
+const StateMachine = require('../../stateMachine/StateMachine')
+const XETHRU_STATE = require('../../SessionStateXethruEnum')
+const ALERT_REASON = require('../../AlertReasonEnum')
+
+const MOVEMENT_THRESHOLD = 40
+const STILLNESS_TIMER = 300
+const DURATION_TIMER = 600
+const INITIAL_TIMER = 200
 const IM21_DOOR_STATUS = require('../../IM21DoorStatusEnum')
-const { getRandomArbitrary, getRandomInt, printRandomIntArray } = require('../../testingHelpers.js')
+const { getRandomArbitrary, getRandomInt, printRandomIntArray } = require('../../testingHelpers')
+const STATE = require('../../stateMachine/SessionStateEnum')
 
 chai.use(chaiHttp)
 chai.use(sinonChai)
+chai.use(chaiDateTime)
 
+const sandbox = sinon.createSandbox()
 const testLocation1Id = 'TestLocation1'
 const testLocation1PhoneNumber = '+15005550006'
-const door_coreID = 'door_particlecoreid'
-const radar_coreID = 'radar_particlecoreid'
+const door_coreID = 'door_particlecoreid1'
+const radar_coreID = 'radar_particlecoreid1'
+
+async function firmwareAlert(coreID, alertReason) {
+  try {
+    await chai
+      .request(server)
+      .post('/api/startChatbot')
+      .send({
+        event: `${alertReason} Alert`,
+        data: 'test-event',
+        ttl: 60,
+        published_at: '2021-06-14T22:49:16.091Z',
+        coreid: coreID,
+      })
+    await helpers.sleep(10)
```

Comment:
Darn, you needed to put `sleep`s back in?

Reply:
Just to make reasoning about thresholds etc easier, not for debugging flakey tests or anything.